frontend/src/metabase/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -1,12 +1,22 @@
-/* eslint "react/prop-types": "warn" */
-
-import PropTypes from "prop-types";
-
 import cx from "classnames";
+import type { ReactNode } from "react";
 
 // NOTE: currently relies on .QueryError CSS selectors residing in query_builder.css
+interface ErrorMessageProps {
```

Comment:
nit and not blocking at all: should we keep the comment near the component? Or did you leave it there to keep it at the top of the file?

Reply:
Good point fixed in 85c282616b7d3dd1beaa0f49df6c00c9f046cf5e